src/tests.builds

```diff
@@ -11,7 +11,8 @@
   <Import Project="$(ToolsDir)VersionTools.targets" Condition="Exists('$(ToolsDir)VersionTools.targets')" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" Exclude="@(TestProjectExclusions)" />
+    <Project Condition="'$(Performance)'!='true'" Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" Exclude="@(TestProjectExclusions)" />
```

Comment:
Yeah, good call.  I had code like that before, but when I did the last rebase I missed this part.  I like your suggestion and will put this in.